portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,30 +49,64 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             List<String> tasks = new ArrayList<>();
             for (Entity entity : results.asIterable()) {
              long id = entity.getKey().getId();
-                String comm = (String) entity.getProperty("content");
-                
-                tasks.add(comm);
+             String nick = (String) entity.getProperty("nickname");
+
+             String comm = (String) entity.getProperty("content");
+
+            if(nick == null){
+                nick = "Anon: ";
+            }
+             comm = nick + comm;
+             tasks.add(comm);
                 
 
             }
 
 			response.setContentType("application/json;");
 			String json = new Gson().toJson(tasks);
-			System.out.println("get: " + tasks);
+			// System.out.println("get: " + tasks);
+
+            //add in nickname to commstream
+            
+
 			response.getWriter().println(json);
 		}
+
+        private String getUserNickname(String id) {
+            DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+            Query query =
+                new Query("UserInfo")
+                    .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+            PreparedQuery results = datastore.prepare(query);
+            Entity entity = results.asSingleEntity();
+            if (entity == null) {
+                return "";
+            }
+            String nickname = (String) entity.getProperty("nickname");
+            return nickname;
+        }
         
         @Override
 		public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 			String text =  getParameter(request, "textarea_field", "");//returns what is in text firld
 			
+            //add in nickname to commstream
+            UserService userService = UserServiceFactory.getUserService();
+            String nickname = getUserNickname(userService.getCurrentUser().getUserId());
+            System.out.println("nickname: " + nickname);
```

Comment:
Is there a reason you are printing this. Debug comments like this can result in privacy concerns.